sdk/Sdk.Analyzers/Sdk.Analyzers.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <MinorProductVersion>1</MinorProductVersion>
```

Comment:
[Current version of Worker.Sdk.Analyzers ](https://www.nuget.org/packages/Microsoft.Azure.Functions.Worker.Sdk.Analyzers/)is 1.0.0. This change will create 1.1.0